src/js/components/recipient/RecipientOverview.jsx

```diff
@@ -5,146 +5,175 @@
+        address = (
+            <td>
+                <div>{recipient.location.streetAddress}</div>
+                <div>{recipient.location.regionalAddress}</div>
+                <div>{recipient.location.fullCongressionalDistrict}</div>
+            </td>
         );
-        if (recipient.location.streetAddress || recipient.location.regionalAddress || recipient.location.fullCongressionalDistrict) {
-            address = (
-                <td>
-                    <div>{recipient.location.streetAddress}</div>
-                    <div>{recipient.location.regionalAddress}</div>
-                    <div>{recipient.location.fullCongressionalDistrict}</div>
-                </td>
-            );
-        }
+    }
 
-        // Format business types
-        let businessTypes = (
-            <td>
+    // Format business types
+    let businessTypes = (
+        <td>
                 Not provided in source system
+        </td>
+    );
+    if (recipient.businessTypes.length > 0) {
+        businessTypes = (
+            <td>
+                {recipient.businessTypes.map((type, i) =>
+                    <div key={i}>{type}</div>)}
             </td>
         );
-        if (recipient.businessTypes.length > 0) {
-            businessTypes = (
-                <td>
-                    {recipient.businessTypes.map((type, i) =>
-                        <div key={i}>{type}</div>)}
-                </td>
-            );
-        }
+    }
 
-        return (
-            <div
-                id="recipient-overview"
-                className="recipient-section recipient-overview">
-                <h2 className="recipient-overview__title">
-                    {recipient.name}
-                    {viewAlternateNames}
-                </h2>
-                <hr className="results-divider" />
-                <div className="recipient-overview__content">
-                    {parent}
-                    <div className="recipient-overview__children">
-                        {label}
-                        {viewChildren}
-                    </div>
-                    <div className="recipient-section__row">
+    const loanTotals = () => (
+        <div className="recipient-section__viz totals-subset">
+            <h5 className="recipient-overview__heading-subset">
+                                Total Face Value of Loans
+                <TooltipWrapper
+                    className="award-section-tt"
+                    tooltipPosition="right"
+                    icon="info"
+                    wide
+                    tooltipComponent={recipientLoanFaceValueInfo} />
+
+            </h5>
+
+            <div className="totals-subset__amount">
+                {recipient.totalLoanFaceValueAmount}
+            </div>
+            <div className="totals-subset__awards">
+                                from <span className="state-overview__total">{recipient.totalLoanTransactions}</span> transactions
```

Comment:
I wonder if we need to adjust your editor... We need to use prettier or something. Very weird that this isn't triggering lint.

cc @ebdabbs